README.md

```diff
@@ -119,7 +119,7 @@ A schema is basically a single data structure. A keyword list, containing two to
     ],
     "my_app.complex_list.*": [
       to: "my_app.complex_list",
-      datatype: [:complex],
+      datatype: :complex,
```

Comment:
Yes, I've removed list from here to distinguish custom types from internal types, but I can add `:complex` here - https://github.com/bitwalker/conform/pull/36/files#diff-869e3aad15833ce4507306803453edf7R7 and it will work for the `[:complex]`